src/controllers/sqlControllers/profileSqlControllers/getProfiles.ts

```diff
@@ -0,0 +1,59 @@
+import { z } from 'zod';
```

Comment:
Have you installed zod? I can't find zod dependency added in the package.json in your PR